### PR TITLE
Update Yelp

### PR DESCRIPTION
**Pull request by fbernhart**
yelp

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/yelp.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...4939 1.111-.3499.02.012 1.596.531 1.596.531 3.5878 1.179 3.7048 1.224 3.8568 1.338.227.186.332.475.285.813h.006zM13.92 12.9594c-.254-.386-.25-.841.012-1.155l.998-1.3589c2.189-2.9839 2.311-3.1409 2.4589-3.2449.256-.171.57-.179.871-.032.869.422 2.6229 3.029 2.7289 4.0289v.034c.029.341-.105.618-.346.784-.164.105-.314.166-4.3929 1.1559-.645.164-1.004.254-1.215.329l.029-.03c-.4039.12-.8539-.074-1.1089-.479zm-2.5039-1.546c-.195.061-.789.245-1.519-.9379 0 0-4.9307-7.7587-5.0467-7.9977-.07-.27.015-.574.255-.82.734-.761 4.7168-1.875 5.7597-1.621.34.088.574.301.656.604.06.335.545 7.5358.615 9.1487.066 1.38-.525 1.565-.72 1.624zm.651 7.8928c-.011 3.7738-.019 3.8998-.081 4.0788-.105.281-.346.469-.681.53-.96.164-3.9668-.946-4.5938-1.69-.12-.164-.195-.328-.21-.493-.016-.12 0-.24.045-.3459.075-.195.18-.345 2.8799-3.5099l.794-.944c.271-.345.75-.45 1.199-.271.436.165.7059.54.6759.945v1.68zm-8.1827-2.414c-.295-.01-.56-.187-.715-.48-.111-.215-.189-.5699-.238-1.0019-.137-1.301.029-3.2639.419-3.8868.183-.285.45-.436.745-.426.195 0 .369.061 4.2288 1.65l1.13.4489c.404.15.654.57.63 1.051-.03.465-.298.824-.694.93l-1.605.51c-3.5898 1.1549-3.7088 1.1849-3.8978 1.1699zm14.9764 7.1048h-.004l-.005.003z"/></svg>
```
Comment (adamrusted) [inline icons/yelp.svg:1]: There seems to be quite a difference between PR (red) and [source `24x24_yelp`](https://www.yelp.com/styleguide/icons) (black)
Any chance you could take another look?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/yelp.svg (hunk 1 above)
Comment (fbernhart) [reply icons/yelp.svg]: @adamrusted Done 👍🏼  I've as well updated the source accordingly.

File: .svglint-ignored.json [hunk 2]
```diff
@@ -155,8 +155,7 @@
...
...-.393l5.608-9.677a.781.781 0 000-.786l-5.608-9.68a.781.781 0 00-.678-.39H6.391zm0 .418H17.56l-5.68 9.836h-6.03l4.712-8.156v-.002l.013-.026.114-.195a.132.132 0 00-.114-.197h-.256l-2.81-.002a.265.265 0 00-.229.13l-4.847 8.391a.539.539 0 00-.03.057H.482l5.592-9.652a.371.371 0 01.317-.184zm11.542.191l5.6 9.67c.032.056.05.12.05.184a.367.367 0 01-.05.184l-5.606 9.677-.002.002-5.688-9.853 5.696-9.864zM7.598 3.633h2.478l-4.63 8.021a.68.68 0 000 .682l4.642 8.033-2.48-.002-4.815-8.318a.106.106 0 010-.106l4.805-8.31zm10.328 1.869a.131.131 0 00-.113.068l-.108.19-.002.002-.015.031h-.002l-1.391 2.465a.261.261 0 000 .26l1.943 3.36a.26.26 0 010 .263l-1.927 3.338a.261.261 0 000 .26l1.4 2.45v.002l.016.028v.002l.11.191a.13.13 0 00.227.002l.112-.195.002-.002.015-.026 3.422-5.92a.525.525 0 000-.525l-3.447-5.955-.018-.031-.11-.192a.13.13 0 00-.114-.066zm.004.71l3.324 5.743a.106.106 0 01-.002.106l-3.299 5.71-1.234-2.162 1.88-3.257.003-.002a.676.676 0 00-.002-.68v-.002l-1.897-3.281 1.227-2.174zM.482 12.21h1.924a.525.525 0 00.026.049l4.857 8.396a.26.26 0 00.227.131l2.814.002h.258a.13.13 0 00.113-.195l-.113-.196v-.001l-.016-.026-4.715-8.16h6.01l5.678 9.836H6.39a.362.362 0 01-.317-.184L.482 12.21ZZ": "XAML",
```
Comment (adamrusted) [inline .svglint-ignored.json:158]: Looks like you've deleted XAML in this file - any chance you could add it back in, just for this commit?

**Review by adamrusted — APPROVED** [review]
LGTM! ⭐

**Pull request merged**